video_xblock/video_xblock.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import logging
+import datetime
```

Comment:
Please reorder imports according to code style
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order